upb/port/undef.inc

```diff
@@ -63,6 +63,7 @@
 #undef UPB_UNPOISON_MEMORY_REGION
 #undef UPB_ASAN
 #undef UPB_ASAN_GUARD_SIZE
+#undef UPB_HAS_FEATURE
```

Comment:
is UPB_HAS_ATTRIBUTE missing from this list?

Reply:
Yes, though it is undef'd [here](https://github.com/protocolbuffers/upb/blob/3d0119a2dd674851107f35f7c039dfcc1b0d2ee4/upb/port/def.inc#L219).

Reply:
let's be consistent and undef these in the same place (your call which)